tube-player/README.md

```diff
@@ -37,29 +40,52 @@ This workshop has been set up to provide you with optional content to allow you
 
 The workshop utilizes the latest Uno Platform features, including:
 
-- [C# Markup](xref:Reference.Markup.GettingStarted)
-- [Extensions](xref:Overview.Features)
-  - [MVUX](xref:Overview.Mvux.Overview)
-    - [Pagination](xref:Overview.Mvux.Advanced.Pagination)
-    - [FeedView](xref:Overview.Mvux.FeedView)
-      - NoneTemplate
-      - ErrorTemplate
-    - [Refit](xref:Overview.Http)
-    - [Region navigation](xref:Overview.Navigation)
-    - [Serialization](xref:Overview.Serialization)
+### Environment setup & Inner loop
+
+- [Hot reload](xref:Uno.Features.HotReload)
+- [Uno.Check](xref:UnoCheck.UsingUnoCheck)
+
+### Cross-platform development
+
+- [Resizetizer](xref:Uno.Resizetizer.GettingStarted)
+  - [UnoIcon](xref:Uno.Resizetizer.GettingStarted#unoicon)
+  - [UnoSplashScreen](xref:Uno.Resizetizer.GettingStarted#unosplashscreen)
+- [Template Wizard](xref:Uno.GettingStarted.UsingWizard)
+
+### Battle-tested features for apps
+
+- [C# Markup](xref:Uno.Extensions.Markup.Overview)
```

Comment:
@nickrandolph, @lukeblevins 
I think having an Uno.Extensions overview on top of this list could be useful here (similar to MVUX overview above)
What do you think ?

Reply:
@lukeblevins not sure I saw the update with the Extensions overview https://platform.uno/docs/articles/external/uno.extensions/doc/ExtensionsOverview.html

Reply:
@agneszitte I somehow forgot that one, despite having authored the extensions overview 🙃